ui/core/proto_utils/utils.ts

```diff
@@ -159,6 +159,26 @@ export const naturalClassOrder: Array<Class> = [
 	Class.ClassWarrior,
 ]
 
+export const ExperimentalRotationSettings = new Set([
+	'preNerfedGargoyle',
+	'desyncRotation',
+])
```

Comment:
Really not a fan of hardcoding these here. We should be able to set a given input as experimental when defining them in a sim's `inputs.ts` files

Reply:
I agree, fixed